### ISSUE TITLE
[BUG] twilio-client seems to break angular build

### ISSUE THREAD

**Issue by Corsico** [twilio/twilio-client.js#139]
**Code to reproduce the issue:**

**Actual behavior:**

twilio-client seems to break angular build

```
ERROR in node_modules/twilio-client/es5/twilio/audiohelper.d.ts(16,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/audiohelper.d.ts(29,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/audiohelper.d.ts(33,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/connection.d.ts(50,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/connection.d.ts(55,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/device.d.ts(106,16): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/device.d.ts(111,16): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/device.d.ts(115,16): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/device.d.ts(119,16): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/device.d.ts(128,16): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/twilio-client/es5/twilio/pstream.d.ts(1,12): error TS2300: Duplicate identifier 'PStream'.
node_modules/twilio-client/es5/twilio/pstream.d.ts(17,18): error TS2300: Duplicate identifier 'PStream'.
node_modules/twilio-client/es5/twilio/pstream.d.ts(17,70): error TS2709: Cannot use namespace 'PStream' as a type.
node_modules/twilio-client/es5/twilio/pstream.d.ts(18,15): error TS2300: Duplicate identifier 'PStream'.
node_modules/twilio-client/es5/twilio/pstream.d.ts(53,16): error TS2709: Cannot use namespace 'PStream' as a type.
```

**Software versions:**

 - [ ] Browser(s):
 - [ ] Operating System:
 - [ ] twilio.js:
 - [ x ] Third-party libraries (e.g., Angular, React, etc.): 
    * "typescript": "^2.9.2",
    * "twilio-client": "^1.6.9"
    * "@angular/cli": "~7.2.1",

**Comment by charliesantos** [comment]
Hi @Corsico , thanks for submitting. Do you know if this has worked on prior versions of Angular or Client SDK?

**Comment by charliesantos** [comment]
Added jira CLIENT-7544

**Comment by charliesantos** [comment]
Fixed in https://github.com/twilio/twilio-client.js/pull/144 and will be included in 1.11. Please watch out for the release.
